ui/tests/unit/components/billing/BillingDialogPaymentMethod.spec.js

```diff
@@ -127,6 +130,21 @@ describe('BillingDialogPaymentMethod', () => {
       expect(wrapper.vm.lockButton).toEqual(false);
     });
 
+    ///////
```

Comment:
When I was thinking about creating the tests, the methods are also within "Data and Props checking".